tsconfig.dev.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Does gulp have no watch mode?

Reply:
I just noticed that gulp supports watch mode.
with gulp watch mode, you can also remove ttypescripts from dependencies.

I'll try to apply it right now.

Reply:
I just tried it, but there is a bug in Transformer, so I don't think it'll be possible to apply it right now.

I'll try to do it in the near future.